tests/integration/test_cli.py

```diff
@@ -58,18 +58,30 @@
 
 @pytest.fixture
 def non_mocked_hosts() -> list:
-    # Let requests go out to localstack/S3
+    """Hosts that shall not be mocked by"""
```

Comment:
```suggestion
    """Hosts that shall not be mocked."""
```

Reply:
"... by httpx" I wanted to say. Since the fixture is not used by ourselves and thus it's a bit magic behavior.